### PR TITLE
Enhancement/36 create user guide

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: doc/user_guide.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Exasol SQL Statement Builder User Guide
+
+You can follow this guide to learn more about how to use the SQL Statement
+Builder. It provides explanation of currently supported features with various
```
Comment (redcatbear) [inline doc/user_guide.md]: "explanations"

File: doc/user_guide.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+code examples.
+
+## Overview
+
+The Exasol SQL Statement Builder abstracts programmatic creation of SQL
+statements and is intended to replace ubiquitous string concatenation solutions
+which make the code hard to read and are prone to errors and security risks.
+
+SQL Statement Builder allows users to create a SQL statement classes using
```
Comment (redcatbear) [inline doc/user_guide.md]: "... create an abstract representation of SQL statements using ..."

File: doc/user_guide.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+[fluent programming][fluent] that can render into a string.
```
Comment (redcatbear) [inline doc/user_guide.md]: " ... programming.

Once you have such an abstract statement, you can for example use the SSB to render the statement into an SQL string."

File: doc/user_guide.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+**Currently supported SQL statements:**
+
+- [CREATE TABLE](#create-table)
+- SELECT
+- INSERT INTO
+
+## CREATE TABLE
+
+The `CreateTable` class of the Sql Statement Builder implements SQL CREATE TABLE
```
Comment (redcatbear) [inline doc/user_guide.md]: of the SQL Statement Builder provides an entry point to defining a `CREATE TABLE` SQL statement.

File: doc/user_guide.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+1. Create an instance of the `CreateTable` class through the `StatementFactory`
+
+  ```java
+  CreateTable createTable = StatementFactory.getInstance().createTable("tableName");
+  ```
+
+2. Create columns with desired data types using fluent programming
+
+  The column order in a final rendered string will be the same as a sequence of
```
Comment (redcatbear) [inline doc/user_guide.md]: "Construct the columns in the order you want them to appear in the created table."

File: doc/user_guide.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+  the method calls.
+
+  For example, create a table with three columns (decimal, char and boolean):
```
Comment (redcatbear) [inline doc/user_guide.md]: `DECIMAL`, `CHAR`, `BOOLEAN`

File: doc/user_guide.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+  ```java
+  createTable.decimalColumn("col_decimal", 9, 0)
+             .charColumn("col_char", 10)
+             .booleanColumn("col_boolean");
+  ```
+
+  Please keep in mind that the column name if required when creating a column.
```
Comment (redcatbear) [inline doc/user_guide.md]: "... name is required ..."

File: doc/user_guide.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+  |``CHAR``                          | ``true``           | `createTable.charColumn("col_char")`                             |
+  |``DATE``                          | ``false``          | `createTable.dateColumn("col_date")`                             |
+  |``DECIMAL``                       | ``true``           | `createTable.decimalColumn("col_dec", 18, 0)`                    |
+  |``DOUBLE PRECISION``              | ``false``          | `createTable.doublePrecisionColumn("col_double_precision")`      |
+  |``INTERVAL DAY TO SECOND``        | ``true``           | `createTable.intervalDayToSecondColumn("col_intdaytosec", 2, 3)` |
+  |``INTERVAL YEAR TO MONTH``        | ``true``           | `createTable.intervalYearToMonthColumn("col_intyeartomonth", 2)` |
+  |``TIMESTAMP``                     | ``false``          | `createTable.timestampColumn("col_timestamp")`                   |
+  |``TIMESTAMP WITH LOCAL TIME ZONE``| ``false``          | `createTable.timestampWithLocalTimeZoneColumn("col_tswithzone")` |
+  |``VARCHAR``                       | ``true``           | `createTable.varcharColumn("col_varchar", 100)`                  |
+
```
Comment (redcatbear) [inline doc/user_guide.md]: :+1:

File: doc/user_guide.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+  You can find more information about the column types in JavaDoc files.
```
Comment (redcatbear) [inline doc/user_guide.md]: "... in the SQL Statement Builder's JavaDoc API description."

File: doc/user_guide.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+
+3. Render the instance of `CreateTable` class
```
Comment (redcatbear) [inline doc/user_guide.md]: This part is identical for all statements. Move it to a separate section and make it generic.

File: doc/user_guide.md (hunk 1 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 3 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 4 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 5 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 6 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 7 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 9 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

File: doc/user_guide.md (hunk 10 above)
Comment (AnastasiiaSergienko) [reply doc/user_guide.md]: fixed

**Pull request merged**